README.md

```diff
@@ -1,7 +1,77 @@
 # Firefox Telemetry Python ETL
 
+[![Build Status](https://travis-ci.org/mozilla/python_mozetl.svg?branch=master)](https://travis-ci.org/mozilla/python_mozetl)
+
```

Comment:
It would be nice to add the coverage badge too:

`[![codecov](https://codecov.io/gh/mozilla/python_mozetl/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/python_mozetl)`